DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -1713,7 +1716,7 @@ public async Task<IEnumerable<DiscordApplicationCommandAutocompleteChoice>> Prov
 		}
 
 		[SlashCommand("help", "Displays command help")]
-		internal async Task DefaultHelpAsync(InteractionContext ctx,
+		internal static async Task DefaultHelpAsync(InteractionContext ctx,
```

Comment:
why are we making this static?

Reply:
VS suggested it

Reply:
I would like to point out that not everything VS Suggests is gold. lol

Reply:
fair